compose-table/src/main/java/org/dhis2/composetable/ui/DataSetTableScreen.kt

```diff
@@ -299,17 +295,9 @@ fun DataSetTableScreen(
                         override fun onTableWidthChanged(tableId: String, newValue: Float) {
                             onTableDimensionResize(tableId, newValue)
                         }
-                    }
+                    },
+                    bottomContent = bottomContent
```

Comment:
Maybe we no longer need the parent column of DataTable?

Reply:
changed!